.github/workflows/_docker_build.yaml

```diff
@@ -76,6 +79,7 @@ jobs:
     needs: prepare-tags
     outputs:
       image: kong/kubernetes-ingress-controller:${{ steps.meta.outputs.version }}
```

Comment:
How does this work? 🤔 We added a separate input - `tag` - but we still provide the same tag to `image` input

Reply:
Ditto, after the change I think `image` is not used after the change.